### ISSUE TITLE
Non-ASCII latin in <u>

### ISSUE THREAD

**Issue by interim-rfc-series-project-manager** [rfc-format/draft-iab-rfc-nonascii-bis#4]
In v3 documents, should latin characters like the ï in naïve be treated as normal text and not spelled out in the unpaginated text output

**Comment by jrlevine** [comment]
For that matter, since we can have UTF-8 in all output formats, do non-ASCII characters need to be spelled out at all?

**Comment by reschke** [comment]
Not necessarily. I believe however, that some form of this element (being optional) is useful so that authors have standard formats to talk about non-ASCII in prose (and to ensure that literal values, code points and expanded names are in sync).

**Comment by levkowetz** [comment]
As an ironic illustration of why it may be a good idea to require something like the &lt;u&gt; element that ensures that the expansion of codepoints are correct, the published RFC7997 in the examples about temperature changes on page 7 and 8 of the PDF displays Unicode symbol U+0394 (Δ) when the surrounding text refers to U+2206, "INCREMENT".

If &lt;u&gt; had been available at the time, and required in order to insert non-ascii characters into the body text, that error could not have happened.

**Comment by reschke** [comment]
I think that's a good argument for having the element; but not necessarily for enforcing it's use (that is something the production center could decide).

FWIW, see <https://www.rfc-editor.org/errata/eid6032> (on the positive side, the errata system seems to grok non-ASCII characters correctly, and apologies for misspelling your last name).

**Comment by jrlevine** [comment]
I noticed that and fixed it in the XML I checked in here a few days ago.

While I can see this as a reason to have a debug option in xml2rfc to spell out non-ASCII characters for production to check, I still don't see why that would go in the published version.

**Comment by levkowetz** [comment]
In order to let the tools make sure the content is right, rather than having to rely on humans checking things like unicode codepoint expansion, I'd say.  The RPC have enough things that humans have to spend time on already.

**Comment by jrlevine** [comment]
If the tools had spelled out GREEK CAPITAL LETTER DELTA (which if it's not clear, I think would be a good idea during the editing process), humans would still have had to notice that wasn't the intended character. Software is great but it'll never be a substitute for proofreading.
